src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/CodeActions/CSharp/DefaultCSharpCodeActionProviderTest.cs

```diff
@@ -347,8 +339,8 @@ private static RazorCodeActionContext CreateRazorCodeActionContext(
             request,
             documentSnapshotMock.Object,
             codeDocument,
-            location,
-            location,
+            absoluteIndex,
+            absoluteIndex,
```

Comment:
nit: a couple of named arguments might help readers a bit here. Same request elsewhere.